tests/sqlparser_common.rs

```diff
@@ -17,6 +17,14 @@
 //! that 1) it's either standard or widely supported and 2) it can be parsed by
 //! sqlparser regardless of the chosen dialect (i.e. it doesn't conflict with
 //! dialect-specific parsing rules).
+//!
+//!
+
+#[macro_use]
+#[path = "utils/mod.rs"]
+mod utils;
+
+use utils::*;
```

Comment:
There's `test_utils.rs` for that already.

Reply:
The actual reason that i don't use the test util is that i want to share the macro between the test , 
And if I understand it correct , the `src` and the the `test` are not considered in the same crate , so i can export macro only via `#[macro_export]` directive, but that mean the macro will be available not only  for the test but to all users of the lib , and that something that i want to avoid (please correct me if i wrong..). 
Anyway - I moved the join and table macros to `test_utils.rs` and change the `utils` dir that i crated to `macros` dir, hope you ok with that. 
btw - why `test_util.rs` located inside the source ?